server/plugin/command_test.go

```diff
@@ -775,31 +775,32 @@ func TestParseCommand(t *testing.T) {
 		{
 			description:        "ParseCommand: subscriptions list command",
 			input:              " /servicenow subscriptions   list  me  all_channels ",
-			expectedAction:     "subscriptions",
+			expectedAction:     constants.CommandSubscriptions,
 			expectedParameters: []string{constants.SubCommandList, constants.FilterCreatedByMe, constants.FilterAllChannels},
 		},
 		{
 			description:        "ParseCommand: subscriptions add command",
 			input:              "/servicenow subscriptions add",
-			expectedAction:     "subscriptions",
+			expectedAction:     constants.CommandSubscriptions,
 			expectedParameters: []string{constants.SubCommandAdd},
 		},
 		{
 			description:        "ParseCommand: subscriptions edit command",
 			input:              "/servicenow subscriptions edit mockID",
-			expectedAction:     "subscriptions",
+			expectedAction:     constants.CommandSubscriptions,
 			expectedParameters: []string{constants.SubCommandEdit, "mockID"},
 		},
 		{
 			description:        "ParseCommand: subscriptions delete command",
 			input:              "     /servicenow       subscriptions      delete     mockID    ",
-			expectedAction:     "subscriptions",
+			expectedAction:     constants.CommandSubscriptions,
 			expectedParameters: []string{constants.SubCommandDelete, "mockID"},
 		},
 		{
-			description:    "ParseCommand: share command",
-			input:          "/servicenow share",
-			expectedAction: constants.CommandSearchAndShare,
+			description:        "ParseCommand: records command",
+			input:              "/servicenow records share",
```

Comment:
at a time i think you can share one record if that's the case I think the command should be  "/servicenow record share" 
if not keep it as it is

Reply:
in the subscription command as well we have kept it as `subscriptions`, so I think it is fine in this way.

Reply:
ok sure but still confirm this once